AppServer_Java/src/com/google/appengine/api/taskqueue/dev/AppScaleTaskQueueClient.java

```diff
@@ -101,6 +99,24 @@ public AppScaleTaskQueueClient() {
         return leaseResponse;
     }
 
+    public TaskQueuePb.TaskQueueDeleteResponse delete(TaskQueuePb.TaskQueueDeleteRequest deleteRequest) {
+        deleteRequest.setAppId(getAppId());
+        Request request = new Request();
+        request.setMethod("Delete");
+        request.setServiceName(SERVICE_NAME);
+        request.setRequestAsBytes(deleteRequest.toByteArray());
+        Response response = sendRequest(request);
+        if (response.hasApplicationError()) {
+            throw new ApiProxy.ApplicationException(
+                response.getApplicationError().getCode(),
+                "TaskQueue Delete operation failed"
```

Comment:
Is there a way to use the protobuf detail field from the response instead of this string? We don't have any for delete calls right now, but we may in the future.